cfn-template/deeplearning.template

```diff
@@ -103,44 +167,57 @@
       "Description" : "The Linux mount point for the EFS volume",
       "Type": "String",
       "MinLength": "1",
-      "Default": "myEFSvolume"
+      "Default": "efs"
```

Comment:
It's better to keep as ``myEFSvolume `` as current README file and auto script commands are running based on this string. Is there a reason it should be changed to efs?